client/landing/jetpack-cloud/sections/backups/main.jsx

```diff
@@ -191,6 +205,7 @@ const mapStateToProps = state => {
 		allowRestore,
 		filter,
 		hasRealtimeBackups: siteSupportsRealtimeBackup( state, siteId ),
+		isConnected: rewindStateRequestStatus !== 'success' || rewind?.state === 'active',
```

Comment:
Can we use a different prop when I think about the isConnected I usually think about the the jetpack connection. 
something like hasServerCredetials would work better here. IMHO.